### ISSUE TITLE
I got higher reconstruction loss in convolutional autoencoder

### ISSUE THREAD

**Issue by csitaula** [keras-team/keras#12512]
I created an CAE and trained Scene15 scene dataset having 15 classes. I made three folders: train, val and test dataset. Each folder contains 15 sub folders to contains images. I trained the model using image data generator. It executed nicely, but the reconstruction loss is always 56% always. Initially it was 69% and it reduced until 56%. I decided to run around 50 epochs. How to avoid this condition? Anybody can help me please?
